ui/lib/kubernetes/addon/components/page/roles.hbs

```diff
@@ -1,7 +1,14 @@
-<TabPageHeader @model={{@backend}} @filterRoles={{true}} @rolesFilterValue={{@filterValue}} @breadcrumbs={{@breadcrumbs}}>
-  <ToolbarLink @route="roles.create" @type="add" data-test-toolbar-roles-action>
-    Create role
-  </ToolbarLink>
+<TabPageHeader
+  @model={{@backend}}
+  @filterRoles={{if @config true false}}
+  @rolesFilterValue={{@filterValue}}
+  @breadcrumbs={{@breadcrumbs}}
+>
+  {{#if @config}}
```

Comment:
Will this also be `null` if there is a permissions error and a user cannot read the config endpoint but can create roles? 🤔

Reply:
Yes good catch!